connect-inject/container_init.go

```diff
@@ -12,9 +12,10 @@ import (
 )
 
 const (
-	InjectInitContainerName = "consul-connect-inject-init"
-	MetaKeyPodName          = "pod-name"
-	MetaKeyKubeNS           = "k8s-namespace"
+	InjectInitCopyContainerName = "consul-connect-inject-init-copy-container"
```

Comment:
Thoughts on naming the new init container?

Reply:
My vote for container names would be for something like:
`consul-connect-copy-init`
`consul-connect-inject-init` and maybe even change the name of the consul-k8s command to `inject-init` rather than `consul-init`.